sql/core/src/main/scala/org/apache/spark/sql/execution/python/ArrowPythonRunner.scala

```diff
@@ -42,7 +43,10 @@ class ArrowPythonRunner(
     argOffsets: Array[Array[Int]],
     schema: StructType,
     timeZoneId: String,
-    conf: Map[String, String])
+    conf: Map[String, String],
+    pythonDataSent: SQLMetric,
+    val pythonDataReceived: SQLMetric,
+    val pythonNumRowsReceived: SQLMetric)
```

Comment:
qq: Why `pythonDataReceived` and `pythonNumRowsReceived` are defined as a variable here ?

Reply:
That's because I need to have pythonDataReceived and pythonNumRowsReceived methods in ArrowPythonRunner to match the definition in the trait PythonArrowOutput.

Reply:
Makes sense, thanks :-)